### PR TITLE
fixed calico link

### PR DESCRIPTION
**Pull request by PascalKu**
Hi there,

The link to the Calico Docs is outdated. I've changed it onto the getting started with kubernetes page of the calico documentation.

Thanks,
Pascal

### CODE REVIEW

File: content/en/docs/concepts/cluster-administration/addons.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ This page lists some of the available add-ons and links to their respective inst
...
 * [Antrea](https://antrea.io/) operates at Layer 3/4 to provide networking and security services for Kubernetes, leveraging Open vSwitch as the networking data plane. Antrea is a [CNCF project at the Sandbox level](https://www.cncf.io/projects/antrea/). 
-* [Calico](https://docs.projectcalico.org/latest/introduction/) is a networking and network policy provider. Calico supports a flexible set of networking options so you can choose the most efficient option for your situation, including non-overlay and overlay networks, with or without BGP. Calico uses the same engine to enforce network policy for hosts, pods, and (if using Istio & Envoy) applications at the service mesh layer.
+* [Calico](https://docs.tigera.io/calico/3.25/getting-started/kubernetes/) is a networking and network policy provider. Calico supports a flexible set of networking options so you can choose the most efficient option for your situation, including non-overlay and overlay networks, with or without BGP. Calico uses the same engine to enforce network policy for hosts, pods, and (if using Istio & Envoy) applications at the service mesh layer.
```
Comment (reetasingh) [inline content/en/docs/concepts/cluster-administration/addons.md:21]: Hi @PascalKu this link points to a specific version of calico (v3.25). Can we instead point it to a non version specific link?

**Comment by dipesh-rawat** [comment]
This seems to be duplicate of https://github.com/kubernetes/website/issues/39538
/triage duplicate

File: content/en/docs/concepts/cluster-administration/addons.md (hunk 1 above)
Comment (bradtopol) [inline content/en/docs/concepts/cluster-administration/addons.md:21]: This seems to be duplicate of https://github.com/kubernetes/website/pull/39538

**Pull request closed without merging**